docs/advanced/storagenetwork.md

```diff
@@ -28,6 +28,20 @@ There are some prerequisites before configuring the Harvester Storage Network se
 - All pods that are attached to Longhorn Volumes should be stopped.
     - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
 
+:::caution
+
+If the Harvester cluster is upgraded from v1.0.3, please check if Whereabouts CNI is installed properly before you move on to the next step. We will always recommend following this guide to check. [Issue 3168](https://github.com/harvester/harvester/issues/3168) describes that the Harvester cluster will not always install Whereabouts CNI properly.
+
+- We should get `ippools.whereabouts.cni.cncf.io` without errors in the following command.
```

Comment:
```suggestion
- Verify the `ippools.whereabouts.cni.cncf.io` CRD exists with the following command.
```

Based on the next point, it seems like the only condition to proceed is if the CRD is on the cluster. 

- If that's the case, I'd suggest being more specific so users know exactly what they're checking for.
- If not, then the introductory clause in the next point should be more generic.